java/flight/flight-sql-jdbc-core/pom.xml

```diff
@@ -107,14 +107,14 @@
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-core</artifactId>
-            <version>3.12.4</version>
```

Comment:
Java 20 was complaining with Mockito v3, so I bumped it to v4 (which still supports JDK 8 unlike v5).